### ISSUE TITLE
[a11y]: The gold card application website CAPTCHA has no accessibility alternative

### ISSUE THREAD

**Issue by komali2** [twa11y/twa11y#36]
### What is the accessibility issue?

The gold card website cannot be logged into by sight-impaired users and users using a screen reader.

The captcha requires viewing text on an image. There's no accessible alternative. 

![2023-07-13_12-57](https://github.com/twa11y/twa11y/assets/8533113/8c265ef1-0a48-4c98-bef5-076382aa9bef)

### How to reproduce the issue?

1. Go to this page: http://...
2. Click on ...
3. The issue is now ...
